pages/common/pueue-completions.md

```diff
@@ -0,0 +1,16 @@
+# pueue completions
+
+> Generates shell completion files for bash, elvish, fish, powershell, zsh.
+> More information: <https://github.com/Nukesor/pueue>.
+
+- Generate completions for bash on Linux:
```

Comment:
Does this only work on Linux? If so, this page should be in the `linux` directory, not the `common` directory. Though bash is supported on Windows (via git bash) and macOS, so I'd suggest removing the `on Linux` bit there.